core/kernel/persistence/smoothsql/class.Resource.php

```diff
@@ -96,7 +96,7 @@ public function getTypes(core_kernel_classes_Resource $resource)
             $uri = $this->getPersistence()->getPlatForm()->getPhpTextValue($row['object']);
             $returnValue[$uri] = $this->getModel()->getClass($uri);
         }
-        
+
```

Comment:
Instead of fixing indent - remove unnecessary line

Reply:
The IDE fixes it automatically, not me, so I did not search for duplicated lines. Anyway I removed the line that was there before :)

Reply:
I know 🙂 
But as this is already in PR - I think it is good chance to fix this 😅